force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -29,7 +29,11 @@ public with sharing class FlowControlLoopsCollections {
      * @return "Hello World!" if x greater than y, otherwise return null.
      */
     public static String helloWorld(Integer x, Integer y) {
-        return null; // Replace null with the variable you used to store the result
+        String helloWorld;
+        if(x>y){
+            helloWorld = 'Hello World!';
+        }
+        return helloWorld; // Replace null with the variable you used to store the result
```

Comment:
Good work. My only suggestion would be to skip the variable assignments.  You normally do not need them if all you do with a value is return it. 
`if(x > y){
     return. 'Hello World';
}`